frontend/src/MuiForms/Dropdown.tsx

```diff
@@ -35,7 +35,10 @@ export default function Dropdown({ label, formContext, value, onChange, options
 
   return (
     <Fragment key={label}>
-      <Typography fontWeight='bold'>{label}</Typography>
+      <Typography fontWeight='bold'>
+        {label}
+        {required && <span style={{ color: 'red' }}>{' *'}</span>}
```

Comment:
Instead of hard coding red we should probably use something like 'palette.error.light'